src/reactors/PriorityOrderReactor.sol

```diff
@@ -64,11 +66,14 @@ contract PriorityOrderReactor is BaseReactor {
     }
 
     /// @notice validate the priority order fields
+    /// - order must not be filled already
     /// - deadline must be in the future
     /// - resolved auctionStartBlock must not be in the future
     /// - if input scales with priority fee, outputs must not scale
     /// @dev Throws if the order is invalid
     function _validateOrder(bytes32 orderHash, PriorityOrder memory order) internal view {
```

Comment:
Seems we don't actually use orderHash here? So can hash the order after calling this to save bit more gas

Reply:
Oh nvm we do use it for cosigner digest

Reply:
Ooo that saved 5k gas!

Reply:
sick!